tuf/settings.py

```diff
@@ -102,6 +102,9 @@
 # the securesystemslib external library.
 DEFAULT_HASH_ALGORITHM = 'sha256'
 
+# The hashing algorithms used to compute file hashes
+FILE_HASH_ALGORITHMS = ['sha256', 'sha512']
```

Comment:
This doesn't apply directly to this PR, but could someone on the TUF team comment on best practices re: configuring `tuf.settings`?

After this PR, Warehouse will probably do something like this:

```python
import tuf.settings

# ...
tuf.settings.FILE_HASH_ALGORITHMS = ['blake2b']
```

is this the intended configuration pattern?

Reply:
I agree it feels wrong, but I fear in some cases it is the only way to configure TUF. We should fix this at some point.